performance-tests/scenarios/04_crud_prompts.yml

```
@@ -0,0 +1,93 @@
+config:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)